cli/src/commands/diffedit.rs

```diff
@@ -77,6 +77,13 @@ pub(crate) struct DiffeditArgs {
         add = ArgValueCompleter::new(complete::revset_expression_mutable),
     )]
     to: Option<RevisionArg>,
+    /// Restrict the diff to these paths
```

Comment:
Even if replace "Restrict" by "Edit only" here, it can't hurt to explicitly say that unmatched paths will be unchanged.

Reply:
This doesn't seem to be addressed